core/tabs/system-setup/arch/virtualization.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+installQEMUDesktop() {
+    if ! command_exists qemu-img; then
+        printf "%b\n" "${YELLOW}Installing QEMU.${RC}"
+        "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm qemu-desktop
+    else
+        printf "%b\n" "${GREEN}QEMU is already installed.${RC}"
+    fi
+    checkKVM
+}
+
+installQEMUEmulators() {
+    if ! "$PACKAGER" -Q | grep -q "qemu-emulators-full "; then
+        printf "%b\n" "${YELLOW}Installing QEMU-Emulators.${RC}"
+        "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm qemu-emulators-full swtpm
+    else
+        printf "%b\n" "${GREEN}QEMU-Emulators already installed.${RC}"
+    fi
+}
+
+installVirtManager() {
+    if ! command_exists virt-manager; then
+        printf "%b\n" "${YELLOW}Installing Virt-Manager.${RC}"
+        "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm virt-manager
+    else
+        printf "%b\n" "${GREEN}Virt-Manager already installed.${RC}"
+    fi
+}
+
+checkKVM() {
+    hardware_avail="$(grep -E 'vmx|svm|0xc0f' /proc/cpuinfo)"
+    kernel_avail="$(zgrep CONFIG_KVM= /proc/config.gz | cut -d '=' -f 2)"
+    modules_avail="$(lsmod | grep kvm)"
+    if [ -z "$hardware_avail" ] || [ -z "$modules_avail" ] || { [ "$kernel_avail" != "m" ] && [ "$kernel_avail" != "y" ]; }; then
```

Comment:
```suggestion
    if [ ! -e "/dev/kvm" }; then
```

We don't need to check all this nonsense. /dev/kvm will not exist if virtualization is disabled or KVM is otherwise unsupported.